aikau/src/main/resources/alfresco/search/AlfSearchList.js

```diff
@@ -468,7 +468,8 @@ define(["dojo/_base/declare",
        */
       updateFilterHash: function alfresco_search_AlfSearchList__updateFilterHash(fullFilter, mode) {
          // Get the existing hash and extract the individual facetFilters into an array
-         var aHash = hashUtils.getHash(),
+         var aHashString = hashUtils.getHashString(),
+             aHash = ioQuery.queryToObject(aHashString),
```

Comment:
It feels a bit wrong to say "this utility isn't doing what I want, so I'll get the bits I need and do it outside the utility instead". Maybe we could just modify the utility getHash and setHash functions to take optional suppressDecoding/suppresssEncoding parameters (respectively), meaning the functionality could be re-used if necessary?